source/models/Loadable/README.md

```diff
@@ -0,0 +1,29 @@
+`Loadable` is a simple helper `Model` that's intended to keep a lodaing state and provide an API similar to FSM.
```

Comment:
Maybe I should know this, but what is `FSM`?

Reply:
@lizlux Oh, sorry, https://en.wikipedia.org/wiki/Finite-state_machine